src/components/main/social_proof/Row.jsx

```diff
@@ -0,0 +1,66 @@
+import styled from 'styled-components';
+
+const RowContainer = styled.div`
+  display: flex;
+  min-width: 75%;
+  padding: 24px 16px;
+  background-color: #1f2a37;
+  border-radius: 8px;
+  border: 1px solid #374151;
+
+  @media (max-width: 768px) {
+  }
+  @media (max-width: 640px) {
+    flex-direction: column;
+    width: 100%;
+    padding: 0;
+  }
+`;
+
+const Stat = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  border: 1px solid #374151;
+
+  ${(props) => ({ borderWidth: props.borderWidth })};
+
+  @media (max-width: 640px) {
+    padding: 12px 0;
+    border: none;
+  }
+`;
+
+const StyledH2 = styled.h2`
+  color: #fff;
+  font-size: 36px;
+`;
+
+const StyledP = styled.p`
+  color: #9ca3af;
+`;
+
+export const Row = () => {
+  return (
+    <RowContainer>
+      <Stat borderWidth="0 1px 0 0">
```

Comment:
tutaj przekazywany jest props borderWidth ze szczegółową instrukcją, ale można by było przekazać boolean (bo widzę tylko 2 wariany, z linią po prawej i bez) i w oparciu o to dodać ten styl w środku, zamiast rozpisywać całe style przy wywołaniu komponentu.

Druga rzecz, tu Stat się powtarza. Można to zmapować :)